packages/Sortable/stories/examples/Basic.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Story } from "storybook/assets/styles/common.styles";
+import { storyStyles } from "../Sortable.stories.helpers";
+import Sortable from "../../src";
+
+const Example = () => {
+  return (
+    <Story css={storyStyles}>
+      <Sortable onChange={() => {}} onRemove={() => {}}>
+        <div>Item One</div>
```

Comment:
what would happen if I do:
```js
<Sortable>
  <MyComponent>one</MyComponent>
  <MyComponent>two</MyComponent>
  <MyComponent>thre</MyComponent>
</Sortable>
```

would that work?

Reply:
It _is_ working, in the story helpers I generate lists of `<Inputs>` that are being used in various stories.

https://github.com/acl-services/paprika/blob/0e07cf3d93d346d3737d96efe0f36092880def91/packages/Sortable/stories/Sortable.stories.helpers.js#L76-L84